From: Jaroslav Kysela <perex@perex.cz>
Date: Tue, 23 Jul 2013 12:50:41 +0000 (+0200)
Subject: local.h: use __kernel_off_t from <linux/types.h>
X-Git-Url: http://git.alsa-project.org/?p=alsa-lib.git;a=commitdiff_plain;h=b3b53a1bec47aaa2baef724cc9e79adadd603caf

local.h: use __kernel_off_t from <linux/types.h>

Also, make the empty definitions __user and __force conditional.

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---

diff --git a/include/local.h b/include/local.h
index 53cb647..f0db188 100644
--- a/include/local.h
+++ b/include/local.h
@@ -32,6 +32,7 @@
 #include <stdarg.h>
 #include <sys/poll.h>
 #include <errno.h>
+#include <linux/types.h>
 #include <linux/ioctl.h>
 
 #include "config.h"
@@ -114,9 +115,12 @@
 #define snd_ctl_tlv		sndrv_ctl_tlv
 
 /* kill and replace kernel-specific types */
+#ifndef __user
 #define __user
+#endif
+#ifndef __force
 #define __force
-#define __kernel_off_t		off_t
+#endif
 
 #include <sound/asound.h>
